File: frontend/src/app/shared/models/poll/poll-result-dto.ts
import { OptionResultDto } from './option-result-dto';
import { VoterDto } from './voter-dto';

export interface PollResultDto {
  pollId: string;
  title: string;
  isAnonymous: boolean;

  optionResults: OptionResultDto[];
  votedUsers: VoterDto[];
}

File: backend/Whale.Shared/Models/Poll/PollResultDTO.cs
using System;
using System.Collections.Generic;

namespace Whale.Shared.Models.Poll
{
	public class PollResultDTO
	{
		public Guid PollId { get; set; }
		public string Title { get; set; }
		public bool IsAnonymous { get; set; }
		public IEnumerable<OptionResultDTO> OptionResults { get; set; }
		public IEnumerable<VoterDTO> VotedUsers { get; set; }
	}
}
